h2o-docs/src/product/data-science/rulefit.rst

```diff
@@ -0,0 +1,125 @@
+
+	**Python only:** To use a weights column when passing an H2OFrame to ``x`` instead of a list of column names, the specified ``training_frame`` must contain the specified ``weights_column``.
+
+	**Note:** Weights are per-row observation weights and do not increase the size of the data frame. This is typically the number of times a row is repeated, but non-integer values are supported as well. During training, rows with higher weights matter more due to the larger loss function pre-factor.
+
+- `distribution <algo-params/distribution.html>`__: Specify the distribution (i.e. the loss function). The options are AUTO, bernoulli, multinomial, gaussian, poisson, gamma, laplace, quantile, huber, or tweedie.
+
+	- If the distribution is ``bernoulli``, the response column must be 2-class categorical.
+	- If the distribution is ``quasibinomial``, the response column must be numeric and binary.
+	- If the distribution is ``multinomial``, the response column must be categorical.
+	- If the distribution is ``poisson``, the response column must be numeric.
+	- If the distribution is ``tweedie``, the response column must be numeric.
+	- If the distribution is ``gaussian``, the response column must be numberic.
+	- If the distribution is ``gamma``, the response column must be numeric.
+	- If the distribution is ``fractionalbinomial``, the response column must be numeric between 0 and 1.
+	- If the distribution is ``negativebinomial``, the response column must be numeric and non-negative.
+	- If the distribution is ``ordinal``, the response column must be categorical with at least 3 levels. 
+	- If the distribution is ``AUTO``,
+
+		- and the response is **Enum** with cardinality = 2, then the family is automatically determined as **bernoulli**.
+		- and the response is **Enum** with cardinality > 2, then the family is automatically determined as **multinomial**.
+		- and the response is numeric (**Real** or **Int**), then the family is automatically determined as **gaussian**.
+
+Interpreting a RuleFit Model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The output for the RuleFit model includes:
+
+- model parameters
+- rule importances in tabular form
+- training and validation metrics of the underlying linear model
+
+Examples
+~~~~~~~~
+
+.. tabs::
+	.. code-tab:: r R
+
+		library(h2o)
+		h2o.init()
+
+		# Import the titanic dataset:
+		f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv"
+		titanic <- h2o.importFile(f)
+
+		# Set the predictors and response; set the factors:
+		response = "survived"
+		predictors <- c("age", "sibsp", "parch", "fare", "sex", "pclass")
+		titanic[,response] <- as.factor(titanic[,response])
+		titanic[,"pclass"] <- as.factor(titanic[,"pclass"])
+
+		# Build and train the model:
+		rf_h2o = h2o.rulefit(y = response, x = predictions, training_frame = titanic, max_rule_length = 10, max_num_rules = 100, seed = 1234)
+
+		# Retrieve the rule importance:
+		print(rf_h2o@model$rule_importance)
```

Comment:
@koniecsveta Would it make sense to show a few more things here, like using `h2o.predict()`?  (Same for Python below).

Reply:
Added .predict() in ecc5047. I am planning to add some more diagnostic tools to rulefit, but this is not done yet. When it will be ready, I would add here to exaplmes also these.